auth/spicedb/policies.go

```diff
@@ -83,7 +85,7 @@ func (pa *policyAgent) AddPolicies(ctx context.Context, prs []auth.PolicyReq) er
 	}
 	_, err := pa.permissionClient.WriteRelationships(ctx, &v1.WriteRelationshipsRequest{Updates: updates, OptionalPreconditions: preconds})
 	if err != nil {
-		return errors.Wrap(errors.ErrMalformedEntity, errors.Wrap(errAddPolicies, err))
+		return errors.Wrap(repoerror.ErrMalformedEntity, errors.Wrap(errAddPolicies, err))
```

Comment:
Errors from the repository or external packages should not leak to the end user. For example here. We should not wrap that error but rather return an error that says what has happened. For example

```suggestion
		return errors.Wrap(repoerror.ErrMalformedEntity, fmt.Errorf("failed to write relaionships"))
```